helm/prometheus-rules/templates/alerting-rules/fairness.rules.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: fairness.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: fairness
+    rules:
+    - alert: FlowcontrolRejectedRequests
+      annotations:
+        description: '{{`Cluster {{ $labels.cluster_id }} .`}}'
+        opsrecipe: flowcontrol-rejected-requests/
```

Comment:
does this exist?

Reply:
hem

Reply:
https://github.com/giantswarm/giantswarm/pull/23547